elliott

```diff
@@ -462,14 +472,22 @@ advisory with the --add option.
 @click.option('--attach', '-a', 'advisory',
               default=False, metavar='ADVISORY',
               help='Attach the builds to ADVISORY (by default only a list of builds are displayed)')
+@click.option("--use-default-advisory", 'default_advisory_type',
+              metavar='ADVISORY_TYPE',
+              type=click.Choice(['image', 'rpm', 'security']),
+              help="Use the default value from ocp-build-data for ADVISORY_TYPE [image, rpm, security]")
 @click.option('--build', '-b', 'builds',
               multiple=True, metavar='NVR_OR_ID',
               help='Add build NVR_OR_ID to ADVISORY [MULTIPLE]')
 @click.option('--kind', '-k', metavar='KIND',
               required=True, type=click.Choice(['rpm', 'image']),
               help='Find builds of the given KIND [rpm, image]')
+@click.option('--base-tag', 'base_tag', metavar="BRANCH",
```

Comment:
Hm. This seems to conflict

```$ ./elliott --group=openshift-4.1 find-builds  --base-tag rhaos-4.1-rhel-8 -k rpm                      
2019-04-30 08:46:05,827 INFO Cloning config data from git@github.com:openshift/ocp-build-data.git
2019-04-30 08:46:08,390 INFO Using branch from group.yml: rhaos-4.1-rhel-7
```

I specified `--base-tag rhaos-4.1-rhel-8` but the message says it's using `branch from group.yml: rhaos-4.1-rhel-7`

Expected?

Reply:
Yes, but admittedly confusing. That's the runtime looking up the default branch/tag. But the `--base-tag` is what actually gets used if specified.

Reply:
You can say `--branch` as a global option but it doesn't get down to the method call. Maybe there's a better way here. But also, we really are talking about the brew tag (plus `-candidate`), not the dist-git branch. The fact they happen to be the same... is confusing.

Reply:
Let's call that out then

```
$ ./elliott --group=openshift-4.1 find-builds  --base-tag rhaos-4.1-rhel-8 -k rpm
2019-04-30 08:46:05,827 INFO Cloning config data from git@github.com:openshift/ocp-build-data.git
2019-04-30 08:46:08,390 INFO Using branch from group.yml: rhaos-4.1-rhel-7
INFO Overriding default branch from group.yml with specified base 'rhaos-4.1-rhel-8'

....
```

Thoughts? Use your best judgement on how to work that precisely.

Reply:
I decided there's no real need for this confusion and we can just use `--branch` and not introduce an override. Updated...